src/components/EditPage.js

```diff
@@ -0,0 +1,39 @@
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+import React from "react";
+import { Link } from "react-router-dom";
+
+const GET_SINGLE_PAGE = gql`
+  query getSingleServicePage($pageSlug: String) {
+    servicePage(slug: $pageSlug, pk: 5) {
```

Comment:
When I left out `, pk: 5` on this line, it returns the error: 
`"Error! GraphQL error: Please provide id or pk"`

I decided to leave it here for now until we know what direction we're going with the backend for these GraphQL queries. I could dig deeper into what we need to do for Joplin to either return a `pk` in the query or for the `id` to actually work as a reference lookup instead of just `pk` and `slug`. 

More here: https://github.com/cityofaustin/techstack/issues/436#issuecomment-398224605